rollup.config.js

```diff
@@ -1,20 +1,39 @@
+import babel from '@rollup/plugin-babel';
 import replace from '@rollup/plugin-replace';
-import commonjs from '@rollup/plugin-commonjs';
 
-export default {
+const commonConfig = {
 	input: 'index.js',
-	output: {
-		'file': __dirname + '/dist/memize.js',
-		format: 'iife',
-		name: 'memize',
-	},
 	plugins: [
-		replace( {
-			preventAssignment: true,
-			values: {
-				'process.env.NODE_ENV': JSON.stringify( process.env.NODE_ENV )
```

Comment:
I didn't apply this setting to the CommonJS build, should it be?

Reply:
I think we probably don't need the `replace` at all anymore. IIRC, it was useful for the pre-built, minified output, so that Uglify/Terser would drop a condition that looked like...


```js
if ('production' === 'test') {
  // ...
}
```

We could probably remove the `NODE_ENV=production` part of the `build:bundle` script in `package.json` as well, since it was related to this.